atd-vzv/src/constants/time.js

```diff
@@ -42,3 +42,8 @@ export const summaryLastYearEndDate = dataEndDate
 
 export const currentYearString = summaryCurrentYearStartDate.slice(0, 4);
 export const prevYearString = summaryLastYearStartDate.slice(0, 4);
+
+// Map time data
+export const mapStartDate = moment().subtract(1, "month").startOf("year");
```

Comment:
I was hung up at first why you would subtract a month, but then I remembered about the January use case. Good stuff. Maybe consider adding a comment here about that intended logic.

Reply:
That is a good idea. This also reminded me to update according to the global time range change in https://github.com/cityofaustin/atd-vz-data/pull/780. I'll update before merging. 👍